src/caw/caw_function.cc

```diff
@@ -182,9 +209,25 @@ void CawFunction::ReadReplys(const std::string& caw_id, KeyValueInterface& kv,
   }
 }
 
-std::vector<std::string> GetHashtags(const std::string& message) {
-  // I will pass a caw text and find tags here
-  return {};
+std::vector<std::string> CawFunction::GetHashtags(const std::string& message) {
+  std::vector<std::string> hashtags; 
+  std::string hashtag = ""; 
+  int i=0; 
+  while (i < message.size()) {
+    if (message[i] == '#') {
+      i++; 
+      while (i < message.size() && std::isalnum(message[i])) {
+        hashtag += message[i++];
```

Comment:
be careful with this since it can be a little confusing to read with multiple operations in one line

Reply:
Yup definitely. Hopefully the test cases cover the edge cases!